dotnet-local.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+if [ -d "bin/Release/dotnet" ]; then
+    ./bin/Release/dotnet/dotnet $@
```

Comment:
use `exec ./bin/Release/dotnet/dotnet "$@"`, which will replace `bash` process with `dotnet` and will properly quote arguments with spaces in them